common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON

```diff
@@ -0,0 +1,1256 @@
+{
+	"info": {
+		"name": "cme_api_v1.2",

```

Comment:
Please change the name to "cme_api_v1" and override the existing file "common/cme_api_postman/v1/cme_api_v1.postman_collection.json"